### PR TITLE
Image sharer issue 5

### PR DESCRIPTION
**Pull request by chris-mason**
Closes #5

---

The following is a checklist to remind you of the steps to take when opening a
new PR. The horizontal line above and everything below can be removed from the
pull request description.

- [ ] I have replaced `X` above with the associated issue number.
- [ ] I have changed the issue's label to "pull request".
- [ ] I have assigned the designated reviewer to this PR.
- [ ] I have set the appropriate label on this PR.
- [ ] I have written clear commit messages ([ref](http://chris.beams.io/posts/git-commit/)).

If needed, please review the document on the
[Image Sharing Project](https://sites.google.com/a/appfolio.com/eng/new-hire-resources/engineering-academy/ropes/image-sharing-project)
for information on how labels should be used.

### CODE REVIEW

**Comment by chris-mason** [comment]
[Issue 5: Image Tags](https://trello.com/c/I94RBxPT/452-issue-5-image-tags)

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,9 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+(1..21).each do |value|
+  Image.create url: "https://assets.pokemon.com/assets/cms2/img/pokedex/full/#{value.to_s.rjust 3, '0'}.png"
```
Comment (infiniteluke) [inline db/seeds.rb:10]: this shouldn't be in issue 5. you may need to rebase on issue 4 and drop this commit

File: db/seeds.rb [hunk 2]
```diff
@@ -8,6 +8,4 @@
 
 (1..21).each do |value|
   Image.create url: "https://assets.pokemon.com/assets/cms2/img/pokedex/full/#{value.to_s.rjust 3, '0'}.png"
-
 end
-
```
Comment (infiniteluke) [inline db/seeds.rb]: we don't want to make style changes that are unrelated to the commit.

File: app/views/application/home.html.erb [hunk 3]
```diff
@@ -9,9 +9,12 @@
   <div>
     <% @images.each do |image| %>
       <row>
-        <div class="col px-md-12">
+        <div class="col px-md-9">
           <img width="400" src=<%= image.url %>>
         </div>
+        <div class="col px-md-3">
+          <h3>Tags: <%= image.tag_list %></h3>
```
Comment (infiniteluke) [inline app/views/application/home.html.erb:16]: this is likely causing an n+1 query unless you're including the tags for each image int he controller already

**Review by infiniteluke — CHANGES_REQUESTED** [review]
A couple changes need. Also, ensure you read the docs on n+1 because I think it's likely that you're querying for each images tags separately.

**Comment by chris-mason** [comment]
Rebased with master and resolved commit history after strange git issues where PR #4 didn't get automatically rebased;

File: db/seeds.rb (hunk 1 above)
Comment (chris-mason) [reply db/seeds.rb]: Rebased and removed

File: db/seeds.rb (hunk 2 above)
Comment (chris-mason) [reply db/seeds.rb]: Rebased and removed

File: app/views/application/home.html.erb (hunk 3 above)
Comment (chris-mason) [reply app/views/application/home.html.erb]: Updated the reference here; Can't use tag list as it still performs duplicate SQL queries even when performing eager loading using "includes"; Updated reference to .tags and issue appears to be resolved

**Review by infiniteluke — APPROVED** [review]

**Pull request merged**